mappings/net/minecraft/entity/boss/dragon/phase/TakeoffPhase.mapping

```diff
@@ -1,2 +1,6 @@
 CLASS net/minecraft/class_1524 net/minecraft/entity/boss/dragon/phase/TakeoffPhase
-	FIELD field_7055 target Lnet/minecraft/class_243;
+	FIELD field_7054 path Lnet/minecraft/class_11;
+	FIELD field_7055 pathTarget Lnet/minecraft/class_243;
+	FIELD field_7056 randomizesPath Z
```

Comment:
This field doesn't seem to involve any randomness.

Reply:
`shouldFindNewPath` works better